modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -307,13 +316,21 @@ def file_exists(cls, file_path: str) -> bool:
                 )
                 from botocore.exceptions import NoCredentialsError
 
-                s3fs = S3FS.S3FileSystem(anon=False)
+                if storage_options is not None:
+                    # make copy
+                    new_storage_options = dict(storage_options)
+                    # avoid keyword duplication problems
```

Comment:
```suggestion
                    new_storage_options = dict(storage_options)
```

nip: I think, it's clear in the code what happens :)